components/alert/src/_mixin.scss

```diff
@@ -25,12 +24,12 @@
 
 @mixin utrecht-alert__icon {
   --utrecht-icon-color: var(--_utrecht-alert-icon-color, var(--utrecht-alert-icon-color));
-
-  grid-area: icon;
+  --utrecht-icon-size: var(--utrecht-alert-icon-size);
+  --utrecht-icon-inset-block-start: var(--utrecht-alert-icon-inset-block-start);
 }
 
 @mixin utrecht-alert__message {
-  grid-area: message;
+  row-gap: var(--utrecht-alert-message-row-gap);
```

Comment:
Refactor als `alert-row-gap`, separated message voor aria-live en een content wrapper waar eventueel actions in kunnen

Reply:
@Robbert n.a.v. conversaties met Jeffrey & Aline hebben we gekozen geen generieke "alert-row-gap" te doen omdat dit verwarring veroorzaakte en niet accuraat aan duid waar de row gap van toepassing is. Er zijn nu twee specifieke row gaps, `message-row-gap` en een `content-row-gap`. Zo kan je de spacing bepalen tussen je message elementen, en je eventuele actions of andere anderelementen.